### PR TITLE
[FIX] 12.0 - sale_timesheet_rounded

### PR DESCRIPTION
**Pull request by enriquemartin**
fixes issue https://github.com/OCA/project/issues/687

### CODE REVIEW

**Review by j-zaballa — APPROVED** [review]
LGTM!

File: sale_timesheet_rounded/models/account_analytic_line.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def read(self, fields=None, load='_classic_read'):
         This affects `account_anaytic_line._sale_determine_order_line`.
         """
         ctx_ts_rounded = self.env.context.get('timesheet_rounding')
-        fields_local = fields.copy() if fields else []
+		fields_local = list(fields) if fields else []
```
Comment (pedrobaeza) [inline sale_timesheet_rounded/models/account_analytic_line.py]: Incorrect indentation

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Remember that now there's no need to bump module version (as it's not recommended for avoiding possible conflicts), as the merge bot does it for you

**Comment by j-zaballa** [comment]
Thanks, @pedrobaeza. Requested changes taken care of. Also changed another instance of `fields.copy()` on `read_group`.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**